apps/ima/behaviours/reset-human-rights-summary.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+  saveValues(req, res, next) {
+    if(req.sessionModel.get('family-members') !== undefined) {
+      if(req.sessionModel.get('human-rights-claim') === 'no' &&
+      JSON.stringify(req.sessionModel.get('family-members').aggregatedValues.length) > 0) {
```

Comment:
what is the reason for the `JSON.stringify() `function? Does it not work if you put just `req.sessionModel.get('family-members').aggregatedValues.length `?

Reply:
I originally did not use the JSON.stringify() locally, which caused an issue reading the length of the array. I've removed it again now and it is working when I test it. Might have been a browser cookie issue.